jenkins-slaves/jenkins-slave-golang/main.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright (c) 2018 Paul Jolly <paul@myitcv.io>. All rights reserved.
```

Comment:
is it possible to provide a different example, so it's not copyrighted?

@pabrahamsson ; thoughts?

Reply:
I chose that example in order to make sure go modules were being handled properly